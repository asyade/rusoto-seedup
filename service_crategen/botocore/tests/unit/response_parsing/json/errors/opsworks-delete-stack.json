{"__type":"ResourceNotFoundException","message":"Unable to find stack with ID foobar"}
