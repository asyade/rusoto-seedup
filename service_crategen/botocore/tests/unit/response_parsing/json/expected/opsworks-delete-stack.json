{
    "ResponseMetadata": {},
    "Error": {
        "Message": "Unable to find stack with ID foobar",
        "Code": "ResourceNotFoundException"
    }
}
